src/main/resources/application.properties

```diff
@@ -1 +1,2 @@
-
+spring.data.mongodb.uri=mongodb+srv://${db.username}:${db.password}@atmosfearscluser.ommn72o.mongodb.net
```

Comment:
Will you create users in the MongoDB on Atlas and give us access to the same database in cloud?

Reply:
Yes, right now there is only one user that can be used by every member of organization. We should discuss if we want to create user per person or leave it like that